### DISCUSSION TITLE
Free.fullspeed.tv links blocked by Avast

### DISCUSSION THREAD

**Discussion by eltoncomp** [iptv-org/community#156] [Show and tell]
Links from the free.fullspeed.tv streamlink service are being blocked by Avast in my computer. When I open these links in VLC media player, Avast shows me this message:
![Captura de tela 2022-01-29 214748](https://user-images.githubusercontent.com/34846333/151682640-1a39b996-6592-400b-ad6e-3a222e88cf0e.png)

**Comment by BellezaEmporium** [comment]
Might be because of misusages some do with the host, otherwise it's basically redirecting links scraped by streamlink, I don't see quite the issue in there.

**Comment by dct-infra** [comment]
Seems like a local or specific avast url blacklist to me.
Can be due to the fact that the url is considered "insecure"
@eltoncomp Try some other non https urls and see if you get the same result.
Also, try to unfold the "Ver detalhes" option, might help us better understand the issue.
Checked on popular blacklists, IP/URL is not listed.
